index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Split that Bill!</title>
+  <link href="https://fonts.googleapis.com/css2?family=Baloo+Thambi+2:wght@400;500;600&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+  <h1>Split that Bill!</h1>
+  <section class='calculator-section'>
+    <form>
+      <label>Bill Amount:</label>
+      <input class='bill-amount' type='text' />
+      <label>Number of People:</label>
+      <input class='num-of-people' type='text' />
+      <label>Rate Your Service:</label>
+      <select default>
+        <option selected='selected'
+          disabled
+          hidden
```

Comment:
Minor UX note: this being hidden was slightly confusing for me when I was clicking around.